consumers/src/unit-test/java/uk/gov/moj/sdt/consumers/SubmitQueryConsumerTest.java

```diff
@@ -131,111 +146,128 @@ public void setUp() {
         submitQueryRequestType = this.createRequestType(submitQueryRequest);
     }
 
+    @Test
+    void getClient() {
+        final String targetApplicationCode = "";
+        final String serviceType = "";
+        final String webServiceEndPoint = "";
+        final long connectionTimeOut = 0L;
+        final long receiveTimeOut = 0L;
+        ITargetAppInternalEndpointPortType portType = submitQueryConsumer.getClient(targetApplicationCode,
+                serviceType, webServiceEndPoint, connectionTimeOut, receiveTimeOut);
+        Assertions.assertTrue(null != portType);
+    }
+
     /**
      * Test to verify submit query consumer does throw expected exception.
      */
-    @Test(expected = TimeoutException.class)
-    public void testSubmitQueryRequestTimeout() {
-        EasyMock.expect(mockTransformer.transformDomainToJaxb(submitQueryRequest)).andReturn(submitQueryRequestType);
+    @Test
+    void testSubmitQueryRequestTimeout() {
+        when(mockTransformer.transformDomainToJaxb(submitQueryRequest)).thenReturn(submitQueryRequestType);
 
         final WebServiceException wsException = new WebServiceException();
         wsException.initCause(new SocketTimeoutException("Timed out waiting for response"));
 
-        EasyMock.expect(mockClient.submitQuery(submitQueryRequestType)).andThrow(wsException);
+        when(mockClient.submitQuery(submitQueryRequestType)).thenThrow(wsException);
 
-        EasyMock.replay(mockTransformer);
-        EasyMock.replay(mockClient);
+        TimeoutException timeoutException = assertThrows(TimeoutException.class, () ->
+                this.submitQueryConsumer.processSubmitQuery(submitQueryRequest, CONNECTION_TIME_OUT, RECEIVE_TIME_OUT));
 
-        this.submitQueryConsumer.processSubmitQuery(submitQueryRequest, CONNECTION_TIME_OUT, RECEIVE_TIME_OUT);
+        assertEquals(timeoutException.getErrorCode(), "TIMEOUT_ERROR");

```

Comment:
Looks like the assertEquals() parameters are the wrong way around - the first should be the expected value and the second the actual value.  The same applies to most of the other occurrences of assertEquals() in this class.

Reply:
For new classes, definitely. For legacy code, we might address this if the class is being edited significantly, otherwise it's low priority.